.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,56 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+    paths: ['r-package/**']
```

Comment:
We might want to do something similar in [.github/workflows/release.yml](https://github.com/posit-dev/shinymedia/blob/chore/r-pkg-setup/.github/workflows/release.yml#L3) to avoid running Python checks unless the python-package files change.

Although there it's slightly more complicated because that workflow relies on tag pushes for publishing. (Otherwise I would have done that now.)